vyper/compiler/output.py

```diff
@@ -159,6 +161,19 @@ def build_interface_output(compiler_data: CompilerData) -> str:
     return out
 
 
+def _get_structs(m: ModuleT, visited: set[ModuleT] = None) -> [StructT]:
```

Comment:
we can use `ModuleT.reachable_imports` here instead of implementing the recursion

Reply:
I kept the recursion because for building the import paths for the output - `a.b.c. struct`.